src/applications/vaos/actions/newAppointment.js

```diff
@@ -18,3 +24,42 @@ export function updateFormData(page, uiSchema, data) {
     data,
   };
 }
+
+export function routeToPage(flow, router, current, action) {
```

Comment:
Maybe `navigationAction` or `routeAction`

Reply:
I want to capture that it's routing/changing the page, I think. `routeToNextPageByAction`? Action may be a bad choice of word in here, because it's not a Redux action.